randovania/games/prime3/logic_database/Bryyo - Fire.txt

```diff
@@ -419,7 +419,7 @@ Extra - asset_id: 13296637487518400273
 > Pickup (Missile Expansion); Heals? False
   * Layers: default
   * Pickup 22; Category? Minor
-  > Above Ground
+  > Event - Gel Pickup
```

Comment:
I want to say this should be left alone and have the event connect to the pickup instead? Not 100% sure but the general pattern is `event -> item`

Reply:
So rearrange the node order to ~`above ground -> event - gel pickup -> pickup (missile exp) -> above ground`?
Will have to move all the logic from `above ground -> pickup (missile)` to `above ground -> event - gel pickup`, right?

Reply:
Correct